intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function() {
+    if (raw_data.keys.length > 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'raw_data' is not in camel case.](https://www.codacy.com/app/tjcsl/ion/file/2620587168/issues/source?bid=3176160&fileBranchId=3299251#l2)